Update IndexedDB platform support (#1042)
apache/cordova-docs: www/docs/en/dev/cordova/storage/storage.md

But unlike LocalStorage, you can create multiple databases, with multiple stores per
database, and its asynchronous API and search indexes provide performance benefits.

IndexedDB is supported by the underlying WebView on all platforms,

> with known limitations on web browser and Windows.

Reviewer: web browser should be plural, shouldn't it?

Author: I am reluctant to make "browser" plural since I am referring to the Cordova `browser` platform and would rather refer to it by proper name, if possible.

Author: What about this idea:

```suggestion
with known limitations on the following platforms:

- browser
- Windows
```

Or do you have any other ideas?

### Web browser limitations

> The actual behavior may depend on which browser is used. There may be differences on Edge and Firefox browsers, for example.

Reviewer: IMO we should not use Edge as an example, because it soon is Chromium based, too. Maybe just use Safari or Chrome instead?

Author: How about this:

```suggestion
The actual behavior may depend on which browser is used. There could be differences between the behavior on Safari and Firefox browsers, for example.
```

But unlike LocalStorage, you can create multiple databases, with multiple stores per
database, and its asynchronous API and search indexes provide performance benefits.

> IndexedDB is supported by the underlying WebView on all platforms,
> with known limitations on the following platforms:

Reviewer: ```suggestion
IndexedDB is supported by the underlying WebView on all platforms, with known limitations on the following platforms:
```

### Disadvantages

- Complex API with nested callbacks.

> - Limited total amount of storage (typically around 5MB).

Author: How about this resolution:

```suggestion
- Total amount of storage may be limited, depending on the platform and web view implementation.
```

Reviewer 2: Looks good to me.
